### PR TITLE
incus: add page

### PR DESCRIPTION
**Pull request by ChuJiani**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 6.0 LTS

### CODE REVIEW

**Comment by spageektti** [comment]
Hi. Thanks for your contribution.

**Comment by ChuJiani** [comment]
Hi, spageektti. Thank you for your revision. I'll do it soon.

File: pages/common/incus.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# incus
+
+> Incus is a modern, secure and powerful system container and virtual machine manager.
```
[the remark this answers is not in the document]
Comment (spageektti) [reply pages/common/incus.md:3]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#wording

**Comment by ChuJiani** [comment]
Rebased to keep up with the latest `main` branch.

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by spageektti — APPROVED** [review]

**Pull request merged**